meson.build

```diff
@@ -24,7 +24,7 @@ endif
 cc = meson.get_compiler('c')
 
 lite_includes = []
-lite_cargs = []
+lite_cargs = ['-DSDL_MAIN_HANDLED']
```

Comment:
`SDL_MAIN_HANDLED` is needed to stop SDL from trying to provide its own WinMain, which we don't need